### DISCUSSION TITLE
特定のキーが反応するけどしない件について

### DISCUSSION THREAD

**Discussion by mutsuki7377** [Yowkees/keyball#412] [General]
お世話になっております。
今回初めての自作キーボートとしてkeyball44を組みました。
その中でAキーのみ、ピンセットでキースイッチのパッドをショートさせると反応するのですが、キースイッチ、キーキャップを組んだうえで押すと反応しません。
何が原因かお分かりになりますでしょうか。
よろしくお願いいたします。

**Comment by mutsuki7377** [comment]
スイッチ用ソケットを交換しようとしたら基盤側のシート？のようなものがくっついてきました。こちらはもう修復不可能でしょうか？
![P_20231104_232515](https://github.com/Yowkees/keyball/assets/149853532/4c96b337-52ad-4077-b98e-4e7c56738508)

![P_20231104_233053](https://github.com/Yowkees/keyball/assets/149853532/3b9c80a2-4fde-476a-99ed-da13b89f9057)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。
下画像の赤点どうしをエナメル線やゴム被覆導線で接続していただければ復活できると思います。
なお、キーソケットの保持力がまったくなくなりますので、キーソケットをボンドや両面テープ等で固定してから作業していただくと良いと思います。

![280473633-4c96b337-52ad-4077-b98e-4e7c56738508](https://github.com/Yowkees/keyball/assets/143698349/6f84a21f-2d0d-4399-b9e5-333c9717e1a4)

**Comment by mutsuki7377** [comment]
不格好ですがなんとかできました
ありがとうございました。

keyball44で打ちました。

![P_20231107_004120](https://github.com/Yowkees/keyball/assets/149853532/313b1936-8d16-4e76-841e-995e7da834cf)
![P_20231107_004133](https://github.com/Yowkees/keyball/assets/149853532/fa49a49e-1a3a-43fa-8f12-db3105e0057a)

**Reply by Ryo624** [reply]
無事に解決していただきまして、ありがとうございます。
ぜひ末永くご愛用していただければ幸いです。
